pages/conference/young-inspirers.vue

```diff
@@ -203,6 +232,128 @@ export default {
+                            full_description: [
+                                'In this talk, I wish to share my Python learning journey from the beginning, when I did not know how to code, and to inspire other young people who are starting out or wish to start. Free courses, workshops and online platforms, competitions all helped me out when I began my journey, and I also hope to share about what made me pursue a learning path in Python and in programming as well as my plans for my learning path, such as learning about machine learning and vision AI. Coding is an essential skill in the future as it is an important part of the digital transformation. It can possibly change the world by changing the way we live and complete tasks more efficiently. Thus, through this sharing, I wish to inspire young people like myself in the past who want to start but do not know where as well as offer tips on problem solving, advice on setting learning goals, and helpful information such as useful learning platforms for them so that they too can learn this essential skill.',
+                            ],
+                            note_link: '',
+                            live_link: 'https://youtu.be/Le8OFKAWsW4',
+                        },
+                        {
+                            tag: 'speech2',
+                            speechdate: 'July 23',
+                            speechtime: '15:00-15:40 (GMT+8)',
+                            title: 'My Quirky Adventures with Python & Tech Communities',
+                            speaker: [
+                                {
+                                    name: 'Carl John Viñas',
+                                    description: [
+                                        'Hi, I am Carl, the usual “stingy developer” (a developer who likes doing optimal solutions using only open-source or free tools). You will usually catch me studying computer science-related stuff, working on geospatial-based applications for work, volunteering in tech organizations like The Programmers\' Guild or Python Philippines, or just working on  "quirky" or social impact projects.{}',
+                                        'For this event, I hope to share with you some of my projects, How Python brings impact to all of us,  how cool tech communities are, and how  Python is really for almost anyone!',
+                                    ],
+                                },
+                            ],
+                            full_description: [
+                                'At this event, I hope to share some of the highlights of my adventures as a "stingy" Python Developer.',
+                                'I will be talking about  quirky and cool social impact projects and my thought process behind them, how] tech communities could be a huge factor to your growth, and how you can leverage Python to make the world a better place and how it can be used by almost anyone!',
```

Comment:
```suggestion
                                'I will be talking about quirky and cool social impact projects and my thought process behind them, how tech communities could be a huge factor to your growth, and how you can leverage Python to make the world a better place and how it can be used by almost anyone!',
```